dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -785,16 +785,18 @@ private void buildRelationObject(Context c, Item item, String value, boolean lef
      *                                          for the right entity
      * @param relationshipType                  The RelationshipType object that we want to check whether it's
      *                                          valid to be added or not
-     * @return                                  A boolean indicating whether the relationship is left or right, will
+     * @return A boolean indicating whether the relationship is left or right, will
      *                                          be false in this case
```

Comment:
In *which* case?  I would say this should be "@return whether the relationship is left".  Except that, if it is *not* left, then what is returned is actually the value of the `left` parameter.  So what does this really mean?

Reply:
The JavaDoc has been updated to clarify the returned boolean